prime-angular-frontend/src/app/modules/adjudication/shared/components/dated-content-table/dated-content-table.component.html

```diff
@@ -18,7 +18,10 @@
                       [disableRipple]="true">{{ item.name | replace: '@idir': '' }}</mat-chip>
           </mat-chip-list>
         </th>
-        <td>{{ item.content }}</td>
+        <td class="flex">
```

Comment:
So we use bootstrap in the app, so we can simplify this class by using it and remove code from scss file
``` class="d-flex justify-content-between" ```